server/routes/overview/overviewController.ts

```diff
@@ -128,7 +129,8 @@ export default class OverviewController {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    const timeline = await this.timelineService.getTimeline(prisonNumber, req.user.token, req.user.username)
+    const allTimelineEvents = await this.timelineService.getTimeline(prisonNumber, req.user.token, req.user.username)
+    const timeline = filterTimelineEvents(allTimelineEvents)
```

Comment:
I think I'd like to see an update to the overview controller test to reflect the fact that we have this new function that fiilters timeline events. But seeing as the existing test still works lets leave it for now. Lets concentrate on getting the core functionality working first 👍 
We can add the test in a separate PR afterwards 👍